### DISCUSSION TITLE
程序会检测到自身吗

### DISCUSSION THREAD

**Discussion by ddzzkunye** [ShirasawaSama/CefDetectorX#52] [General]
一个检测webview的软件自己是用webview写的,  并且安装包和vscode差不多大
这波啊,是  
webview太好用了.jpg

**Comment by itldg** [comment]
![image](https://github.com/ShirasawaSama/CefDetectorX/assets/13432299/7f21fd30-90f5-4ac7-8dbe-0af09378cd8c)
会的
